user/serializers.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from user.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
```

Comment:
The `min_length` for the password is set to 5. It's generally recommended to have a longer minimum password length for better security, typically at least 8 characters.